src/main/java/kitchenpos/shared/error/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,24 @@
+package kitchenpos.shared.error;
+
+import kitchenpos.products.ui.error.ProductErrorCode;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import java.util.NoSuchElementException;
+
+@RestControllerAdvice
+public class GlobalExceptionHandler {
+
+    @ExceptionHandler(NoSuchElementException.class)
+    public ResponseEntity<Object> handleNoSuchElementException(NoSuchElementException e) {
+        ProductErrorCode errorCode = ProductErrorCode.PRODUCT_NOT_FOUND;
+        return ResponseEntity.status(HttpStatus.NOT_FOUND)
```

Comment:
ProductErrorCode에 HttpStatus속성을 추가하여 응집성을 높여 보는 건 어떠세요?
```suggestion
        return ResponseEntity.status(errorCode.status)
```

Reply:
ProductErrorCode가 구현하고 있는 인터페이스에 HttpStatus 속성을 추가하도록 변경하겠습니다!